modules/java_api/src/test/java/org/intel/openvino/compatibility/TestsSuite.java

```diff
@@ -1,8 +1,7 @@
-package tests.compatibility;
+package org.intel.openvino.compatibility;
```

Comment:
Why tests share main package? I hope they are not included in the final JAR?

Reply:
Sorry, let me rephrase, I hope that tests are not a part of package and cannot be imported in the user's app.

Reply:
The test classes share the same package as the classes they test to provide access to package-scope classes during testing. However, to keep the tests separate from the main package during compilation and assembly, they are stored in a different physical directory- `src/test/java/` than the main package `src/main/java`. The Gradle java plugin compiles and assembles only the main package in the final JAR.